### PR TITLE
Create custom dashboard title

### PR DESCRIPTION
**Pull request by allonsy**
address #2632 
For the user dashboard, I've appended the user display name
For the organization dashboard, I've appended the org's display name to the beginning
For example, it would say: "user Dashboard - Gogs: Go Git Service"
Currently, the word 'Dashboard' is capitalized, I can decapitalize it in code or by adding a locale string if you think that would be better

Thanks for all your hard work! This really is a wonderful product!
Let me know what you think!

### CODE REVIEW

File: routers/user/home.go [hunk 1]
```diff
@@ -83,11 +83,11 @@ func retrieveFeeds(ctx *middleware.Context, ctxUserID, userID, offset int64, isP
 }
 
 func Dashboard(ctx *middleware.Context) {
-	ctx.Data["Title"] = ctx.Tr("dashboard")
+  ctxUser := getDashboardContextUser(ctx)
+	ctx.Data["Title"] = ctxUser.DisplayName() + " " + ctx.Tr("dashboard")
```
Comment (Unknwon) [inline routers/user/home.go]: Maybe we could remove `" " + ctx.Tr("dashboard")` completely?

Comment (allonsy) [inline routers/user/home.go]: Sure!
The only thing reason why I left it in was largely to differentiate it from the user/org's profile page but I'm fine either way

Comment (allonsy) [inline routers/user/home.go]: I'll leave it up to you, If you want me to remove it, I'd be happy to

Comment (Unknwon) [inline routers/user/home.go]: Only you can see the dashboard for your account, and the page UI is almost 99% different, I think it's not a problem.

**Comment by allonsy** [comment]
Agreed, I made the change

File: routers/user/home.go [hunk 2]
```diff
@@ -83,11 +83,11 @@ func retrieveFeeds(ctx *middleware.Context, ctxUserID, userID, offset int64, isP
 }
 
 func Dashboard(ctx *middleware.Context) {
-	ctx.Data["Title"] = ctx.Tr("dashboard")
+  ctxUser := getDashboardContextUser(ctx)
```
Comment (Unknwon) [inline routers/user/home.go]: Please do `gofmt`.

**Comment by allonsy** [comment]
Sorry, my IDE indentation was acting funny. Ran gofmt and fixed the indentation

**Comment by Unknwon** [comment]
Thanks!

**Pull request merged**